src/series.py

```diff
@@ -0,0 +1,29 @@
+def fib_func(n):
+    """Return fib value"""
+    if n == 1:
+        return 1
+    elif n == 0:
```

Comment:
Logically, the n == 0 condition should come before the n == 1.

Additionally, you can use if instead of elif.   The only time the elif is hit is when the first condition will be hit.   Similarly, the only time the second if will be hit is when the first is not true due to a return..